windows/src/main/csharp/ch/cyberduck/ui/Views/TransfersWindow.xaml.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright(c) 2002 - 2024 iterate GmbH. All rights reserved.
+// https://cyberduck.io/
+// 
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+using System;
+using System.Windows;
+
+namespace ch.cyberduck.ui.Views
+{
+    public partial class TransfersWindow : Window
+    {
+        public TransfersWindow()
+        {
+            InitializeComponent();
+        }
+
+        protected override Size MeasureOverride(Size availableSize)
+        {
+            ToolBar.Measure(new(double.PositiveInfinity, double.PositiveInfinity));
```

Comment:
Force window to not be smaller than the minimum required size by dockpanel.
The make-shift Toolbar doesn't support item collapsing/overflow.